client/driver/docker.go

```diff
@@ -97,12 +100,33 @@ func (d *DockerDriver) Fingerprint(cfg *config.Config, node *structs.Node) (bool
 	return true, nil
 }
 
-// We have to call this when we create the container AND when we start it so
-// we'll make a function.
-func createHostConfig(task *structs.Task) *docker.HostConfig {
-	// hostConfig holds options for the docker container that are unique to this
-	// machine, such as resource limits and port mappings
-	return &docker.HostConfig{
+func containerBinds(alloc *allocdir.AllocDir, task *structs.Task) ([]string, error) {
```

Comment:
Is there a reason this isn't a method of the Docker Driver? It seems unlikely to be shared